src/Couchbase.Lite.Tests.Android/Couchbase.Lite.Tests.Android.csproj

```diff
@@ -101,23 +101,20 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="FluentAssertions">
-      <Version>4.19.4</Version>
+      <Version>5.8.0</Version>
     </PackageReference>
     <PackageReference Include="Newtonsoft.Json">
       <Version>12.0.1</Version>
     </PackageReference>
     <PackageReference Include="xunit">
-      <Version>2.3.1</Version>
+      <Version>2.4.1</Version>
     </PackageReference>
     <PackageReference Include="xunit.runner.devices">
-      <Version>2.2.0</Version>
+      <Version>2.3.3</Version>
     </PackageReference>
   </ItemGroup>
-  <ItemGroup Condition=" '$(Configuration)' == 'Debug' ">
-    <PackageReference Include="Couchbase.Lite.Enterprise.Debug" Version="2.6.0-b0022" />
-  </ItemGroup>

```

Comment:
This should not be deleted.  This will pull the debug version of the package for a debug build.  If you want to change the version you will need to edit by hand.